internal/config/config.go

```diff
@@ -343,6 +385,19 @@ func (c *Config) Validate(skips ...SkipValidation) error {
 			errs = append(errs, fmt.Sprintf("invalid monitor staleness: %s can't be negative", c.Monitor.Staleness))
 		}
 	}
+	{ // Kubernetes
+		if c.Kube.Config != "" && !ptr.Deref(c.Kube.Enabled, false) {
+			errs = append(errs, fmt.Sprintf("%s supplied but %s set to false", KubeConfigFlag, KubernetesFlag))
+		}
+		if c.Kube.Node != "" && !ptr.Deref(c.Kube.Enabled, false) {
```

Comment:
```suggestion
		if c.Kube.Node != "" && !ptr.Deref(c.Kube.Enabled, false) {
```
shouldn't we insist that this isn't empty? other wise it will watch all nodes - right?

Reply:
changed